sequence_task_3/student/include/impl.h

```diff
@@ -11,4 +12,18 @@
  *
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+std::string reverseWords(const std::string& input)
+{
+    std::istringstream iss(input);
+    std::deque<std::string> words;
+    std::copy(std::istream_iterator<std::string>(iss), std::istream_iterator<std::string>(),
+        std::front_inserter(words));
+
+    std::ostringstream oss;
+    std::copy(words.begin(), words.end(), std::ostream_iterator<std::string>(oss, " "));
+    std::string result { oss.str() };
+    if (!result.empty()) {
+        result.resize(result.size() - 1);
```

Comment:
pop_back() is more appropriate method for this task

Reply:
Done!